### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**
Following #16098

| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | N/A |
| License | MIT |
| Doc PR | N/A |

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ public function write($sessionId, $data)
                 $mergeStmt->bindParam(':data', $encoded, \PDO::PARAM_STR);
                 $mergeStmt->bindValue(':time', time(), \PDO::PARAM_INT);
 
-                //Oracle has a bug that will intermitently happen if you
+                //Oracle has a bug that will intermittently happen if you
                 //have only 1 bind on a CLOB field for 2 different statements
                 //(INSERT and UPDATE in this case)
```
Comment (xabbuh) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php]: Can you make the whitespace change here that @javiereguiluz suggested in https://github.com/symfony/symfony/pull/16098#discussion_r41088015?

Comment (pborreli) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandler.php]: done, thanks

**Comment by xabbuh** [comment]
The rest of the changes from #16098 only applies to Symfony 2.7?

**Comment by pborreli** [comment]
@xabbuh well it doesn't apply to 2.3, I guess I have to test manually each changes

**Comment by xabbuh** [comment]
@pborreli thanks

:+1: 

Status: Reviewed

**Comment by fabpot** [comment]
Thank you @pborreli.

**Pull request merged**